client/benches/tps/utils.rs

```diff
@@ -173,11 +174,15 @@ impl MeasurerUnit {
         ));
         self.client.submit_blocking(register_me)?;
 
-        let can_burn_my_asset = PermissionToken::new("can_burn_user_asset".parse()?)
-            .with_params([("asset_id".parse()?, asset_id.clone().into())]);
+        let can_burn_my_asset = PermissionToken {
+            definition_id: "CanBurnUserAsset".to_owned(),
```

Comment:
> Just an idea. We can make Validator to be a library too. That way we don't have to use strings like that.

I think you are on the right track. A better idea would be to put all permission tokens into a separate crate. That crate can then be imported by validator or the client library or any other that wishes to create tokens

Reply:
> I think you are on the right track. A better idea would be to put all permission tokens into a separate crate. That crate can then be imported by validator or the client library or any other that wishes to create tokens.

I like this approach, because client will be able to inspect tokens.

Reply:
opened #3640